submission/programs/tlf-primary.r

```diff
@@ -4,8 +4,10 @@
 # The path variable needs to be defined by using example code below
 #
 # nolint start
-# path = list(adam = "path/to/esub/analysis/adam/datasets")    	# Modify path to the actual location
-# path$output = "."                                             # Output saved in current folder
+# path <- list(
+#   adam = "path/to/esub/analysis/adam",      # Modify path to the adam location
+#   output = "path/to/esub/analysis/output",  # Modify path to the adam location
```

Comment:
the comment here, should say #Modify path to the output location.

Reply:
Hi, @kaz462 .  I went ahead and updated the comment so that I can merge and run the scripts and continue testing.